src/routes/HomePage/HomePage.js

```diff
@@ -5,14 +5,17 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faCat, faStar, faPaw } from "@fortawesome/free-solid-svg-icons";
 import PetsService from '../../services/petsService';
 import PetCard from '../../components/PetCard';
-import { useNavigate } from 'react-router';
 import * as logUtils from '../../components/Utils/Logger';
+import { Link } from 'react-router-dom';
+
+const { HOSTNAME } = require('../../config/hostname.config');
 const log = logUtils.getLogger();
 
+
 export default function HomePage() {
 
   const [pets, setPets] = useState(null);
-  const navigate = useNavigate();
+  const [petNews, setpetNews] = useState(null);
```

Comment:
This is nitpicking but can we change setpetNews to setPetNews?

Reply:
changed!